analytics_data_api/v0/views/learners.py

```diff
@@ -40,15 +46,7 @@ class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns viewing data for each segment of a video.  For each segment,
```

Comment:
It's especially funny because I was copying and pasting this as an example to build my own off of, and noticed that it was wrong. I could have ended up making the same mistake!